specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json

```diff
@@ -216,6 +269,24 @@
           }
         }
       }
+    },
+    "RoutingForNumber": {
```

Comment:
naming: Should this be called `RoutesForNumber` (returns a collection with multiple routes)

Reply:
Changed the name, thanks for the remark